repository/f.json

```diff
@@ -1508,6 +1508,17 @@
 				}
 			]
 		},
+		{
+			"name": "FontSizePerView",
```

Comment:
You can actually have spaces in the name, makes it easier to read. Just a suggestion.